.circleci/config.yml

```diff
@@ -23,16 +23,18 @@ jobs:
             echo 'export CPPFLAGS=-I$HOME/.local/include' >> $BASH_ENV
             echo 'export CXXFLAGS=-I$HOME/.local/include' >> $BASH_ENV
             echo 'export LOCAL_PREFIX=$HOME/.local' >> $BASH_ENV
+            cat /boot/config-`uname -r`
+            dmesg
```

Comment:
Why?

Reply:
For printing kernel config and clock source.